docs/02_cleos/02_how-to-guides/how-to-vote.md

```diff
@@ -1,30 +1,47 @@
-## Goal
+## Overview
 
-Vote for a block producer
+This how-to guide provides instructions on how to vote for block producers.
 
 ## Before you begin
 
-* Install the currently supported version of `cleos`
+* Install the latest version of `cleos`.
 
-* Ensure the reference system contracts from `eosio.contracts` repository is deployed and used to manage system resources
+* Ensure the [reference system contracts](https://developers.eos.io/manuals/eosio.contracts/latest/build-and-deploy) are deployed and used to manage system resources.
 
 * Understand the following:
-  * What is a block producer
-  * How does voting works
+  * What is a [block producer](https://developers.eos.io/welcome/latest/protocol/consensus_protocol#11-block-producers).
+  * How does [voting](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/vote) work.
 
-* Unlock your wallet
+* Unlock your wallet.
 
-## Steps
+## Command Reference
 
-Assume you are going to vote for blockproducer1 and blockproducer2 from an account called `eosiotestts2`, execute the following:
+See the following reference guides for command line usage and related options for the `cleos` command:
+
+* The [`system voteproducer prods` cleos sub-command](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/system/system-voteproducer-prods).
+
+## Procedure
+
+The following steps show:
+
+1. How to vote for blockproducer1 and blockproducer2 from an account called `eosiotestts2`:
 
 ```sh
 cleos system voteproducer prods eosiotestts2 blockproducer1 blockproducer2
 ```
 
-This should produce similar output as below:
+Where:
+
+* eosiotestts2 = the account that votes.
+* blockproducer1 and blockproducer2 = the accounts receiving the votes. The number of accounts receiving the votes can vary from one to multiple. Maximum default number of block producers one account can vote for is 30.
```

Comment:
Might want to use backticks for the arguments to be consistent with the other how-tos above.